includes/wizards/class-reader-revenue-wizard.php

```diff
@@ -204,7 +210,7 @@ public function register_api_endpoints() {
 		// Update Donations settings.
 		register_rest_route(
 			NEWSPACK_API_NAMESPACE,
-			'/wizard/' . $this->slug . '/donations/',
+			' / wizard / ' . $this->slug . ' / donations / ',
```

Comment:
This does not seem correct 🤔

Reply:
Weird, not sure where that came from—probably a PHPCS misconfiguration in my IDE?

Reply:
Fixed in 773c6d9